### ISSUE TITLE
Add location should 

### ISSUE THREAD

**Issue by reneelynn** [Crypton/statusapp#178]
Related but not quite dupe of #176 

Once you enable location services, the "near x location" shown above the text box in my first image 

![img_0926](https://cloud.githubusercontent.com/assets/3867179/10521182/f69162e6-7333-11e5-8a42-6b152f80c143.PNG)

does not show for subsequent posts even though those subsequent posts do indeed add a location.

![img_0923](https://cloud.githubusercontent.com/assets/3867179/10521077/6ba6892c-7333-11e5-84d4-9a71dcc5b729.PNG)
![img_0924](https://cloud.githubusercontent.com/assets/3867179/10521078/6bbb9a38-7333-11e5-8bc2-056da998742c.PNG)

Suggest having that "near [location]" text above every status text entry as long as location is enabled.

iOS6 
Kloak build: 0.6.0pre 20151013

**Comment by daviddahl** [comment]
Ok! excellent bug:)

**Comment by daevski** [comment]
Yeah, good find. I currently see "near Unknown" every time I try to post (inconsistent with this bug) - but haven't tried to remedy the fact that it says Unknown. I will post a bug around that if I can't figure it out.

**Comment by daviddahl** [comment]
The "near unknown" is due to the fuzziness of the GPS data. We literally package up "most" GPS -> location pairs into the app so we don't ever talk to external APIs for this information. It is not perfect and there is a whole giant project in making this data consumption more accurate and more efficient.

**Comment by daviddahl** [comment]
We purposely do not give exact location - the GPS is made fuzzy out to about 2 - 3? miles from where you are.

**Comment by daviddahl** [comment]
I think what we need here is an option for "Include location in each post" (that can be overridden at save() time)

**Comment by reneelynn** [comment]
so in the newest android version it looks like we're only adding location for each specific post. I'm okay with this new behavior I'm seeing and it renders the original report in this ticket moot, unless you want to change the ticket to focus on your last comment.
